src/main/java/it/gov/pagopa/spontaneouspayment/config/MappingsConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package it.gov.pagopa.spontaneouspayment.config;
+
+
+import org.modelmapper.Converter;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.convention.MatchingStrategies;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import it.gov.pagopa.spontaneouspayment.entity.Organization;
+import it.gov.pagopa.spontaneouspayment.mapper.ConvertOrganizationModelToOrganizationEntity;
+import it.gov.pagopa.spontaneouspayment.model.OrganizationEnrollmentModel;
+
+@Configuration
+public class MappingsConfiguration {
+
+    @Bean
+    public ModelMapper modelMapper() {
+        ModelMapper mapper = new ModelMapper();
+        mapper.getConfiguration().setMatchingStrategy(MatchingStrategies.STRICT);
```

Comment:
```suggestion
        mapper.getConfiguration().setMatchingStrategy(MatchingStrategies.STANDARD);
```

Reply:
see comment below

Reply:
it is not advisable to use the standard conversion because there is a risk of losing control of the mappings and could have errors that are difficult to understand